src/main/java/com/flightstats/hub/app/HubMain.java

```diff
@@ -54,6 +55,7 @@
 public class HubMain {
 
     private static final DateTime startTime = new DateTime();
+    private final String hubType = HubProperties.getProperty("hub.type", "aws");
```

Comment:
This feels like something that wants to be an enum. Which, for bonus points, an enum.valueOf() will also blow up if you pass it a value it doesn't know.

Reply:
I agree, I will time box a solution that does not use HubProperties, that class is deprecated and slated for change in another story. Changing how it is used here is a pattern that would warrant changes in integral startup processes of the hub

Reply:
Longer term, it'd be nice to have something more type safe than HubProperties. All those strings is asking for a tpyo issue, but I know that's scattered throughout the code. However, in this case I was thinking smaller scale, something like this:
```
private final HubType hubType = HubType.valueOf(HubProperties.getProperty("hub.type", "aws"));
```